consts.py

```diff
@@ -0,0 +1,11 @@
+POST = "POST"
```

Comment:
Good idea, consider moving this file to `utils` or another suitable folder.

Reply:
fixed